e2e/wdio/core/tests/breadcrumb.e2e-spec.ts

```diff
@@ -0,0 +1,81 @@
+import { BreadcrumbPo } from '../pages/breadcrumb.po';
+import {
+    getElementArrayLength,
+    isElementClickable,
+    refreshPage,
+    scrollIntoView, waitForElDisplayed, waitForPresent
+} from '../../driver/wdio';
+
+import {
+    disabledLinksState,
+    linksActiveState,
+    linksExample, linksHoverState
+} from '../fixtures/testData/breadcrumb.tags';
+import { checkElementActiveState, checkElementHoverState } from '../../helper/assertion-helper';
+import { breadcrumbDisabledLinks, breadcrumbLinks } from '../fixtures/appData/breadcrumb-contents';
+
+
+describe('Breadcrumb test suite:', function() {
+
+    const breadcrumbPage: BreadcrumbPo = new BreadcrumbPo();
+    const { links, disabledLinks } = breadcrumbPage;
+
+    beforeAll(() => {
+        breadcrumbPage.open();
+    }, 1);
+
+    afterEach(() => {
+        refreshPage();
+        waitForPresent(links);
+    }, 1);
+
+    it('should check links', () => {
+        const linksLength = getElementArrayLength(links);
+        for (let i = 0; i < linksLength; i++) {
+            scrollIntoView(links, i);
+            expect(isElementClickable(links, i)).toBe(true);
+        }
+    });
+
+    it('should check disabled links', () => {
+        const disabledLinksLength = getElementArrayLength(disabledLinks);
+        for (let i = 0; i < disabledLinksLength; i++) {
+            scrollIntoView(disabledLinks, i);
+            waitForElDisplayed(disabledLinks, i);
```

Comment:
I took a look at the examples for this one. I think it's not that they are disabled links, but that it's showing the current page (it's a breadcrumb afterall), so current page breadcrumb is without a link at all. Since you are doing visual regression for the example blocks (which contain this element), I think you could remove this test. Or rename from 'disabled' to 'current page breadcrumb' + remove scrollintoview + change waitForElDisplayed to expect(isElementDisplayed, i).toBe(true) [this one has scrollintoview already inside it].  --  but i think visual regression covers it